### PR TITLE
Add discardZeroSign optional parameter to rounding functions

### LINKED ISSUE #22: $numbers.round returns -0 with small negative numbers
Example:  `round(2)(-3.552713678800501e-15)` returns `-0`.  

The expected returned value should be `0`.

### PR DESCRIPTION
**Pull request by davidecaruso**
Fixes #22.

In addition to add new unit tests to cover 0 sign issue, I also updated some duplicated test cases.

### CODE REVIEW

File: src/number.test.ts [hunk 1]
```diff
@@ -196,16 +228,16 @@ describe('number', () => {
...
+      expect(ceil(2)(10.005)).toBe(10.01)
+      expect(ceil(2)(100.005)).toBe(100.01)
+      expect(ceil(2)(1000.005)).toBe(1000.01)
+      expect(ceil(2)(10000.005)).toBe(10000.01)
+      expect(ceil(2)(100000.005)).toBe(100000.01)
+      expect(ceil(2)(1000000.005)).toBe(1000000.01)
+      expect(ceil(2)(10000000.005)).toBe(10000000.01)
+      expect(ceil(2)(100000000.005)).toBe(100000000.01)
+      expect(ceil(2)(1000000000.005)).toBe(1000000000.01)
+      expect(ceil(2)(10000000000.005)).toBe(10000000000.01)
```
Comment (xzhavilla) [inline src/number.test.ts:240]: Oops! Thanks!

File: src/number.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ const approx =
         `${mantissa}e${(exponent ? Number(exponent) : 0) - digits}`,
       Number,
       (n) => Math.sign(a) * Math.abs(n), // Helps when rounding -0.
+      (n) => (n === 0 && discardZeroSign ? Math.abs(n) : n), // If true, -0 -> 0
```
Comment (xzhavilla) [inline src/number.ts:17]: I think line 16 is used to preserve signed zero. You should be able to get the same result discarding your line and making mine conditional.

Comment (davidecaruso) [reply src/number.ts:17]: I think it's being quite difficult to read, for that reason is treated as a case apart, but I also like one-line functions, so ok!

Comment (xzhavilla) [reply src/number.ts]: Your choice then, if you prefer reverting it's fine!

**Pull request merged**